miniteltest/expectations.go

```diff
@@ -179,6 +180,16 @@ func (ts *TestServer) Wait(max time.Duration) bool {
 	}
 }
 
+// ExpectDone returns a non-nil error when unfulfulled expectations are present in
+// the test server.
+func (ts *TestServer) ExpectDone() error {
```

Comment:
If the `TestServer` takes a `testing.T` or an interface that provides `Fatalf` (from testing.T) then you can actually fail the test, which would be nicer.